stl/inc/format

```diff
@@ -707,15 +715,23 @@ public:
                   using _Td = remove_const_t<_Ty>;
                   // doesn't drop const-qualifier per an unnumbered LWG issue
                   using _Tq = conditional_t<_Formattable_with<const _Ty, _Context>, const _Ty, _Ty>;
+                  if constexpr (_Formattable_with_non_const<_Tq, _Context>) {

```

Comment:
This ctor is exposition-only per [[format.arg]/10](https://eel.is/c++draft/format.arg#10) but is public in MSVC STL...
Perhaps we should consider making it private (in another PR, together with non-Standard public ctors of `basic_format_context`).